docs/en/integrations/data-ingestion/dbms/dynamodb/index.md

```diff
@@ -0,0 +1,137 @@
+---
+sidebar_label: DynamoDB
+sidebar_position: 10
+slug: /en/integrations/dynamodb
+description: ClickPipes allows you to connect ClickHouse to DynamoDB.
+keywords: [clickhouse, dynamodb, connect, integrate, table]
+---
+
+import CloudNotSupportedBadge from '@theme/badges/CloudNotSupportedBadge';
+import ExperimentalBadge from '@theme/badges/ExperimentalBadge';
+
+# CDC from DynamoDB to ClickHouse
+
+This page covers how set up CDC from DynamoDB to ClickHouse using ClickPipes. There are 2 components to this integration:
+1. The initial snapshot via S3 ClickPipes
+2. Real-time updates via Kinesis ClickPipes
+
+
+Data will be ingested into a `ReplacingMergeTree`. This table engine is commonly used for CDC scenarios to allow update operations to be applied. More on this pattern can be found in the following blog articles:
+
+* [Change Data Capture (CDC) with PostgreSQL and ClickHouse - Part 1](https://clickhouse.com/blog/clickhouse-postgresql-change-data-capture-cdc-part-1?loc=docs-rockest-migrations)
+* [Change Data Capture (CDC) with PostgreSQL and ClickHouse - Part 2](https://clickhouse.com/blog/clickhouse-postgresql-change-data-capture-cdc-part-2?loc=docs-rockest-migrations)
+
+## 1. Set up Kinesis Stream
+
+First, you will want to enable a Kinesis stream on your DynamoDB table to capture changes in real-time. We want to do this before we create the snapshot to avoid missing any data.
+Find the AWS guide located [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/kds.html).
+
+![dynamodb-kinesis-stream.png](../images/dynamodb-kinesis-stream.png)
+
+## 2. Create the snapshot
+
+Next, we will create a snapshot of the DynamoDB table. This can be achieved through an AWS export to S3. Find the AWS guide located [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/S3DataExport.HowItWorks.html).
+You will want to do a "Full export" in the DynamoDB JSON format.
```

Comment:
How does S3 ClickPipe file size limitation of 1Gb play with the `full export`? https://clickhouse.com/docs/en/integrations/clickpipes/object-storage#limitations

Reply:
Sadly AWS doesn't publish this, but [a stackoverflow answer from an AWS employee](https://stackoverflow.com/questions/77539088/what-is-max-size-of-a-single-zip-object-stored-in-s3-bucket-as-a-result-of-a-dy#:~:text=DynamoDB%20optimizes%20the%20export%20procedure,files%20usually%20hover%20around%20128MB.) says:

> Although there's no specified maximum file size, individual files usually hover around 128MB.

So I think we should be good here.